src/pkg/signature/notary/model/model.go

```diff
@@ -15,7 +15,6 @@
 package model
 
 import "github.com/theupdateframework/notary/tuf/data"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380